Code/PgSQL/rdkit/adapter.cpp

```diff
@@ -738,6 +738,13 @@ extern "C" const char *MolInchi(CROMol i, const char *opts) {
   std::string inchi = "InChI not available";
 #ifdef RDK_BUILD_INCHI_SUPPORT
   const ROMol *im = (ROMol *)i;
+  // Older versions of the InChI code returned and empty string for molecules
+  // without atoms. This changed with 1.07, but we'll keep doing an empty string
+  // here
+  if (!im->getNumAtoms()) {
+    inchi = "";
```

Comment:
std::string inchi;

Reply:
Doh! You can see the "well, this is the same as the other one... I don't need to compile it!" thought process